src/client/components/Sorting/Sorting.styles.css

```diff
@@ -0,0 +1,28 @@
+select {
```

Comment:
avoid using tag selectors, try to use css classes instead.

Reply:
Thank you @younesmln . I did all points.

Reply:
Thank you **@younesmln** . I did all points.

Reply:
@zmzade are sure you pushed your new changes ? because I can't see any difference since the last time no new commits